mix.exs

```diff
@@ -29,7 +29,7 @@ defmodule Plug.MixProject do
   # Configuration for the OTP application
   def application do
     [
-      applications: [:crypto, :logger, :mime],
+      extra_applications: [:logger],
```

Comment:
:crypto needs to be in extra because it is OTP. :)

Reply:
It is included in plug_crypto.

Reply:
Oh, doh.